docs/deployment/README.md#cumulus-instance-sizing

```diff
@@ -422,9 +422,20 @@ Notes on specific variables:
 
 Consider [the sizing of your Cumulus instance](#cumulus-instance-sizing) when configuring your variables.
 
-#### Configure the Thin Egress App
+#### Choose a distribution API
```

Comment:
This header level doesn't seem correct.     It's a sub topic under `### Configure and deploy the `cumulus-tf` root module`, but then the new sub sections ` ## Configure the Thin Egress App` and ` ## Configure the Thin Egress App` are at a higher level than the surrounding sections.

Reply:
Yup, you're totally right. It seems like it should be:

`### Configure and deploy the cumulus-tf root module`
`#### Choose a distribution API`
`#### Configure the Thin Egress App`
`#### Configure the Cumulus Distribution API (optional)`
`#### Deploy`

Where everything under that main heading is at the same level. Choosing your distribution API is a concrete step on equal footing with configuring one or the other in my mind.

https://github.com/nasa/cumulus/pull/2353/commits/d632a8bd5332b8b7c1ab10e20490ece54154812c